doc/source/ray-air/tuner.rst

```diff
@@ -0,0 +1,140 @@
+.. _air-tuner:
+
+Hyperparameter Tuning
+=====================
+The Ray AIR Tuner API is the recommended way to launch hyperparameter tuning jobs with Ray Tune.
```

Comment:
I think we should phrase this differently. In here it sounds like Ray AIR Tuner is a wrapper over Ray Tune, but actually they should be one and the same. 

How about saying "The Tuner API is the recommended of tuning hyperparameters in Ray AIR."?